hail/python/hailtop/hailctl/config/cli.py

```diff
@@ -85,8 +176,30 @@ def set(parameter: str, value: str):
         config.write(f)
 
 
+def get_config_variable(incomplete: str):
+    from hailtop.config import get_user_config  # pylint: disable=import-outside-toplevel
+
+    config = get_user_config()
+
+    elements = []
+    for section_name, section in config.items():
+        for item_name, value in section.items():
+            if section_name == 'global':
+                path = item_name
+            else:
+                path = f'{section_name}/{item_name}'
+            elements.append((path, value))
+
+    config_items = {config.name: config.help_msg for config in config_variables()}
+
+    for name, current_value in elements:
+        if name.startswith(incomplete):
+            help_msg = config_items.get(name)
+            yield (name, help_msg)
```

Comment:
Can this whole function be deleted now and we use `complete_config_variable` in `unset` and `get`?

Reply:
It depends on what we want the behavior to be for unsetting and getting variables that exist already that are not known in the validations. The way it's implemented now is we load all possibilities from the existing configuration. You're proposing to show all known possibilities even if they haven't been set before and don't exist in the configuration.

Reply:
Ah! I missed that distinction. In that case I'm good with listing what's already in the configuration